release-notes/1.19.1.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.19.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+
+* The `anbox-cloud-appliance status` command now lists the version of the application that you are running.
+* The following snaps will be held from being automatically refreshed. They are refreshed only when the charms are upgraded.
+    - `ams`
+    - `ams-node-controller`
+    - `anbox-cloud-dashboard`
+    - `anbox-stream-agent`
+    - `anbox-stream-gateway`
+    - `lxd`
+* You can now see when an application or its version was created using the `CreatedAt` field listed when you run `amc application show`.
+* Android security updates for September 2023 (see Android Security Bulletin - September 2023 for more information).
+* The Android WebView has been updated to 116.0.5845.93.
```

Comment:
Our webview auto build job just bumped up and updated the webview to 116.0.5845.163 yesterday.
```
commit 9cf7b77dd5d750a41ece7b2bbcdadffdf1fdc8c5 (stable-1.19)
Author: Anbox Cloud Job Build Bot <anbox-cloud-jb-bot@canonical.com>
Date:   Sun Sep 3 09:38:17 2023 +0000

    feat(prebuilts): update Android Webview to 116.0.5845.163
```